extensions/Worker.Extensions.CosmosDB/release_notes.md

```diff
@@ -4,6 +4,6 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.CosmosDB <version>
+### Microsoft.Azure.Functions.Worker.Extensions.CosmosDB 4.13.0
 
-- <entry>
```

Comment:
Should the `Microsoft.Extensions.Azure` upgrade be documented as well?

Reply:
Fixed in latest iteration.